src/dai.rs

```diff
@@ -163,7 +164,7 @@ impl Amount {
     }
 
     pub fn symbol(self) -> String {
```

Comment:
You don't need to pass `self` here. Also, why not using the constant directly? (especially that you made the const `pub`).

Reply:
I remove the function for now and just use the `const` exposed by the assets, can potentially use `enum Symbol` introduced by https://github.com/coblox/nectar/pull/48/files

Reply:
introduced enum